plugins/octopus-deploy/src/api/index.ts

```diff
@@ -106,13 +108,25 @@ export class OctopusDeployClient implements OctopusDeployApi {
     return responseJson;
   }
 
-  private async getApiUrl(projectId: string, releaseHistoryCount: number) {
+  private async getApiUrl(opts: {
+    projectId: string;
+    spaceId?: string;
```

Comment:
Could have made sense to make these into `project: ProjectReference` instead, and make `ProjectReference` public (same above). But it's a matter of taste, feel free to ignore

Reply:
I like that suggestion so I've made this change as well.